tt: add macOs binary and additional archives to release Assets

better0fdead opened the pull request:
Added macOs binary, archive with submodules and archive with deps. to the release assets.

Closes #367
Closes #349

LeonidVas commented on .github/workflows/publish.yml:
```diff
@@ -64,16 +57,140 @@ jobs:
+          goreleaser release ${{ steps.set-goreleaser-flags.outputs.GORELEASER_FLAGS }} --skip-publish --config .goreleaser_linux.yml
+
+      - name: Upload packages artifacts
+        uses: actions/upload-artifact@v2
```
Why not v3?

better0fdead replied:
changed

LeonidVas commented on .github/workflows/publish.yml:
```diff
@@ -64,16 +57,140 @@ jobs:
+          name: packages-darwin
+          path: ./dist-prebuilt/packages-darwin
+
+      - name: Remove not-needed files
```
How did you get the list of files that need to be removed?

better0fdead replied:
I have looked what files goreleaser produce, than removed not-needed. I don't think that hardcode will cause troubles.

LeonidVas reviewed (changes requested):
Hi! Thank you for the patch.
- Please use `macOS` or `macos` not `macOs` (in filenames, comments ...)
- Commit title: "tt: ..." -> "cd: ...."
- Split the addition of binaries for macOS and tar.gz for gentoo into separate commits.

better0fdead commented:
> Hi! Thank you for the patch.
> 
>     * Please use `macOS` or `macos` not `macOs` (in filenames, comments ...)
> 
>     * Commit title: "tt: ..." -> "cd: ...."
> 
>     * Split the addition of binaries for macOS and tar.gz for gentoo into separate commits
> 
>     * Can we move all `goreleaser` stuff to separate directory?

Changed
Changed
Changed
Yes, moved

LeonidVas reviewed (approved):
LGTM
